Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -714,12 +784,61 @@ referralView push state =
       ] <> FontStyle.tags TypoGraphy
     ]
 
+nammaSafetyView :: forall w. (Action -> Effect Unit) -> HomeScreenState -> PrestoDOM (Effect Unit) w
+nammaSafetyView push state =
+  linearLayout
+  [ width WRAP_CONTENT
+  , height WRAP_CONTENT
+  , visibility if (any (_ == state.props.currentStage) ) [RideAccepted, RideStarted, ChatWithDriver] then VISIBLE else GONE
+  , stroke $ "1," <> Color.grey900
+  , margin (MarginHorizontal 16 16)
+  , cornerRadius 20.0
+  , background Color.white900
+  , accessibility ENABLE
+  , accessibilityHint $ "Safety + : Button : Select to view S O S Options"
+  , gravity CENTER_VERTICAL
+  , clickable true
+  , padding (Padding 12 8 12 8)
+  ][ imageView 
+    [ imageWithFallback $ fetchImage FF_ASSET "ny_ic_namma_safety"
+    , width $ V 24
+    , height $ V 24
+    , margin $ MarginRight 4
+    ]
+  , textView $ 
+    [ width WRAP_CONTENT
+    , height WRAP_CONTENT
+    , color Color.blue900
+    , accessibility DISABLE
+    , text $ getString NAMMA_SAFETY
+    ] <> FontStyle.body1 TypoGraphy
+  ]
+
+sosView :: forall w. (Action -> Effect Unit) -> HomeScreenState -> PrestoDOM (Effect Unit) w
+sosView push state =
+  linearLayout
+  [ width WRAP_CONTENT
+  , height WRAP_CONTENT
+  , visibility if (any (_ == state.props.currentStage) ) [RideAccepted, RideStarted, ChatWithDriver] then VISIBLE else GONE
+  , margin (MarginHorizontal 16 16)
+  , cornerRadius if os == "IOS" then 20.0 else 32.0
+  , clickable true
+  , onClick push $ const OpenEmergencyHelp
+  ][ imageView 
+    [ imageWithFallback $ fetchImage FF_ASSET "ny_ic_sos"
+    , width $ V 50
+    , height $ V 50
+    , accessibilityHint $ "S O S : Button : Select to view S O S options"
+    , accessibility ENABLE
+    ]
+  ]
+
 liveStatsDashboardView :: forall w. (Action -> Effect Unit) -> HomeScreenState -> PrestoDOM (Effect Unit) w
 liveStatsDashboardView push state =
   linearLayout
     [ width WRAP_CONTENT
     , height WRAP_CONTENT
-    , visibility if state.data.config.dashboard.enable && (state.props.isReferred || state.props.hasTakenRide) && state.props.currentStage == RideStarted then VISIBLE else GONE
+    , visibility GONE
```

Comment:
can we remove the view if it is not required.

Reply:
For now not required, will be added in future so left it in gone